Mathlib/MeasureTheory/Constructions/AddChar.lean

```diff
@@ -0,0 +1,26 @@
+/-
+Copyright (c) 2024 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.Algebra.Group.AddChar
+import Mathlib.MeasureTheory.MeasurableSpace.Defs
+
+/-!
+# Measurable space instance for additive characters
+
+This file endows `AddChar A M` with the trivial sigma-algebra.
```

Comment:
```suggestion
This file endows `AddChar A M` with the discrete measurable space structure whenever `A` and `M` 
are themselves discrete measurable spaces.
```
(I think "trivial" is unclear here – it could refer either to the top or the bottom element of the lattice of sigma-algebras – so I suggest "discrete".)